# struct regs definition conflicts with the struct regs 
# declaration from /usr/include/sys/dtrace.h.
# This patch has been submitted to the community.
--- bfd/aoutf1.h	2014-10-14 00:32:02.000000000 -0700
+++ bfd/aoutf1.h	2015-08-04 11:47:24.254878877 -0700
@@ -251,7 +251,7 @@
 /* But the reg structure can be gotten from the SPARC processor handbook.
   This really should be in a GNU include file though so that gdb can use
   the same info.  */
-struct regs
+struct sparc_regs
 {
   int r_psr;
   int r_pc;
@@ -285,7 +285,7 @@
   int c_magic;				/* Corefile magic number.  */
   int c_len;				/* Sizeof (struct core).  */
 #define	SPARC_CORE_LEN	432
-  struct regs c_regs;			/* General purpose registers -- MACHDEP SIZE.  */
+  struct sparc_regs c_regs;		/* General purpose registers -- MACHDEP SIZE.  */
   struct external_exec c_aouthdr;	/* A.out header.  */
   int c_signo;				/* Killing signal, if any.  */
   int c_tsize;				/* Text size (bytes).  */
@@ -308,7 +308,7 @@
   int c_magic;				/* Corefile magic number.  */
   int c_len;				/* Sizeof (struct core).  */
 #define	SOLARIS_BCP_CORE_LEN	456
-  struct regs c_regs;			/* General purpose registers -- MACHDEP SIZE.  */
+  struct sparc_regs c_regs;		/* General purpose registers -- MACHDEP SIZE.  */
   int c_exdata_vp;			/* Exdata structure.  */
   int c_exdata_tsize;
   int c_exdata_dsize;
